moderation/utils.py

```diff
@@ -12,3 +12,11 @@ def clear_builtins(attrs):
             new_attrs[key] = attrs[key]
 
     return new_attrs
+
+
+def is_sites_framework_enabled():
+    """
+    Check project's settings to see if the optional Sites framework is installed.
+    """
+    from django.conf import settings
+    return True if "django.contrib.sites" in settings.INSTALLED_APPS else False
```

Comment:
Can we make it as `return "django.contrib.sites" in settings.INSTALLED_APPS`? And add type hint for return value as `bool`?

Reply:
Also, could you please rebase and then I believe we can merge that changes also to `master` branch.
Thanks for your activity :)

Reply:
Done. :)